text/0000-add-equality-operators.md

```diff
@@ -0,0 +1,63 @@
+- Start Date: 2019-12-08
+- Relevant Team(s): (fill this in with the [team(s)](README.md#relevant-teams) to which this RFC applies)
+- RFC PR: https://github.com/emberjs/rfcs/pull/560
+- Tracking: (leave this empty)
+
+# Adding Equality Operators to Templates
+
+## Summary
+
+Add new built-in template `{{eq}}` and `{{neq}}` helpers to perform basic equality operations in templates, similar to those included in `ember-truth-helpers`.
+
+This RFC is a subset of the changes proposed in #388.
+
+## Motivation
+
+It is a very common need in any sufficiently complex Ember app to perform some equality operations and often the most convenient place to do it is right in the templates.
+Because of that, [ember-truth-helpers](https://github.com/jmurphyau/ember-truth-helpers) is one of the most installed addons out there, either directly by apps or indirectly by
+other addons that those apps consume.
+
+The fact that `ember-truth-helpers` is so popular is a good signal that this it is filling a perceived gap in Ember's functionality.
+
+A second reason is that it might help make Ember more approachable to newcomers that have some experience in other frameworks.
+Most if not all web frameworks have some way of comparing values in the templates and it's surprising that Ember requires an third party package to perform
+even the most basic operations.
+
+
+## Detailed design
+
+Add `{{eq}}` and `{{neq}}` helpers.
+
+#### `{{eq}}`
+Binary operation. Throws an error if not called with exactly two arguments.
+Equivalent of <arg1> === <arg2>
+This is identical to the `eq` helper in `ember-truth-helpers`
+
+#### `{{neq}}`
```

Comment:
This feels bizarre to me. Firstly, I prefer using `(not (eq` over specifically `not-eq`/`neq`. Secondly, I don't particularly like this abbreviation (`neq`) and I'd prefer `notEq` here.